### PR TITLE
New package: konquest-18.12.0

### PR DESCRIPTION
**Pull request by laserbat**

### CODE REVIEW

File: srcpkgs/konquest/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'konquest'
+pkgname=konquest
+version=18.12.0
+revision=1
+build_style=cmake
+makedepends="qt5-devel qt5-svg-devel kconfig-devel kcoreaddons-devel kcrash-devel kdbusaddons-devel kdoctools-devel kguiaddons-devel
```
Comment (jnbr) [inline srcpkgs/konquest/template]: please break long lines at ca. 80 characters

**Comment by laserbat** [comment]
Builds are failing due to this issue: https://github.com/void-linux/void-packages/issues/5916

File: srcpkgs/konquest/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'konquest'
+pkgname=konquest
+version=18.12.0
+revision=1
+build_style=cmake
+makedepends="qt5-devel qt5-svg-devel kconfig-devel kcoreaddons-devel
+kcrash-devel kdbusaddons-devel kdoctools-devel kguiaddons-devel
```
Comment (maxice8) [inline srcpkgs/konquest/template]: every new line on the same variable must be prefixed with a whitespace char

Comment (laserbat) [reply srcpkgs/konquest/template]: Thank you, fixed

**Comment by Chocimier** [comment]
extra-cmake-modules need to be placed in hostmakedepends. Some other packages needed for cross build should be added to hostmakedepends too, see other kde templates.

makedepends can be much shorter. Run `xmindep qt5-devel … libkdegames-devel` then change makedepends to its output.

**Comment by Chocimier** [comment]
Still do not cross build. Some tools in hostmakedepends are required. Please test with `./xbps-src -a armv7l pkg konquest` if you can.

**Comment by laserbat** [comment]
@Chocimier yeah, I missed another tool! Sorry for wasting your time and thank you for providing tips!

**Comment by Hoshpak** [comment]
Please follow the capitalization according to Manual.md:

```
If you've imported a new package use "New package: <pkgname>-<version>".
```

Also please rebase on top of the current master to resolve the conflict regarding shlibs.

**Comment by laserbat** [comment]
@Hoshpak done! Thank you

**Comment by laserbat** [comment]
@Hoshpak yeah, I saw qt5-devel in konsole hostmakedepends so I assumed it was the correct way to do things.

**Pull request merged**